src/Page/Auth/Signup.jsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from "react-router-dom";
+
+const Signup = () => {
+  return (
+    <>
+      <div className="p-4 bg-white ">
+        <form action="" className="">
```

Comment:
You have empty attributes on some tags, remove them

Reply:
Yes I forgot this